app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,42 @@
+class CommentsController < ApplicationController
+
+  def create
+    session[:return_to] = request.referer
```

Comment:
Nice use of request.referer to save you from hard coding the path! I'm going to take this page out of your book!